.github/workflows/main.yaml

```diff
@@ -15,21 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout source code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: "0"
 
-      - name: The API should not change once published
-        run: |
-          if ! git diff --quiet origin/master -- pkg/apis/sparkoperator.k8s.io/v1beta1; then
-            echo "sparkoperator.k8s.io/v1beta1 api has changed"
-            false
-          fi
-          if ! git diff --quiet origin/master -- pkg/apis/sparkoperator.k8s.io/v1beta2; then
-            echo "sparkoperator.k8s.io/v1beta2 api has changed"
-            false
-          fi
-
```

Comment:
what's the reason for removing this part?

Reply:
there are changes in formatting in golang 1.19 -> 1.22, which caused changes at these locations. cannot ignore formatting changes, cause format check is failing then

Reply:
But the removed code here is `git` command and shell script, not related to Go lang. Or am I missing something?

Reply:
this lines check if there were changes in `pkg/apis/sparkoperator.k8s.io/v1beta1` and as you can see from PR, there are formatting changes there

Reply:
@AndrewChubatiuk Now that the PR is merged, could you create another PR to add this part back?